src/features/classification/security-controls/SecurityControlsItem.js

```diff
@@ -1,49 +1,30 @@
 // @flow
 import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, type MessageDescriptor } from 'react-intl';
```

Comment:
nit: Is this a common pattern for imports? @jstoffan I thought we have the stand on separating imports and type should be on its own